applications/GeoMechanicsApplication/CMakeLists.txt

```diff
@@ -93,4 +96,11 @@ install(TARGETS KratosGeoMechanicsApplication DESTINATION libs )
 
 # Define custom targets
 set(KRATOS_KERNEL "${KRATOS_KERNEL};KratosGeoMechanicsCore" PARENT_SCOPE)
-set(KRATOS_PYTHON_INTERFACE "${KRATOS_PYTHON_INTERFACE};KratosGeoMechanicsApplication" PARENT_SCOPE)
\ No newline at end of file
+set(KRATOS_PYTHON_INTERFACE "${KRATOS_PYTHON_INTERFACE};KratosGeoMechanicsApplication" PARENT_SCOPE)
+
+project(KratosGeoFlow)
+set(KratosGeoFlowSources ${CMAKE_CURRENT_SOURCE_DIR}/custom_workflows/GeoFlow.cpp ${CMAKE_CURRENT_SOURCE_DIR}/custom_workflows/GeoFlow.h)
+add_executable(KratosGeoFlow ${KratosGeoFlowSources})
+target_link_libraries(KratosGeoFlow PUBLIC KratosCore KratosGeoMechanicsCore KratosStructuralMechanicsCore ${CMAKE_DL_LIBS})
+set_target_properties(KratosGeoFlow PROPERTIES LINKER_LANGUAGE CXX)
+install(TARGETS KratosGeoFlow DESTINATION libs)
```

Comment:
Shouldn't this be installed directly in the root (`DESTINATION .`)?

Since we don't have executables this is kinda new for us, but I wouldn't mix the binaries with the lib folder. We can also create a dedicated 'bin' inside Kratos if needed.